packages/tools/tests/test/audioV2/utils/abstractSound.utils.ts

```diff
@@ -3,7 +3,10 @@ import { GetPulseCounts } from "./audioV2.utils";
 
 import { Page } from "@playwright/test";
 
+export const EvaluateTestAsync = async <T>(page: Page, soundType: SoundType, testFn: ({ soundType }: { soundType: SoundType }) => Promise<T>) => {
+    return await page.evaluate(testFn, { soundType });
```

Comment:
I am going to resolve it right after writing it, but - isn't it just better to use this line in the tests instead of writing a function to do that?

Reply:
I don't like having the `{ soundType }` argument added for each test between the eval and the assertions. I think it's easier to read the tests with moved up higher out of the way.